tcms/testplans/templates/testplans/get.html

```diff
@@ -326,6 +327,32 @@ <h2 class="card-pf-title">
 
 </div>
 
+<div class="modal fade" id="default-tester-modal" tabindex="-1" role="dialog" aria-labelledby="default-tester-modal" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true" aria-label="Close">
+            <span class="pficon pficon-close"></span>
+          </button>
+          <h4 class="modal-title" id="default-tester-modal-title">{% trans "Default Tester" %}</h4>
```

Comment:
`Default Tester` -> `Default tester` to minimize variations in translation strings

```suggestion
          <h4 class="modal-title" id="default-tester-modal-title">{% trans "Default tester" %}</h4>
```